packages/firebase/index.d.ts

```diff
@@ -7617,6 +7617,22 @@ declare namespace firebase.storage {
     md5Hash?: string | null;
   }
 
+  /**
+   * An error returned by the Firebase Storage SDK.
+   */
+  interface FirebaseStorageError {
+    name: string;
+    code: string;
+    message: string;
+    serverResponse: null | string;
```

Comment:
Who adds `serverResponse` to the error object?
In https://github.com/firebase/firebase-js-sdk/pull/3946, I'm moving template data under a new field `customData` to avoid overwriting existing fields, should we do the same for `serverResponse`?